examples/rtp-send/main.go

```diff
@@ -6,8 +6,10 @@ import (
 	"os"
 
 	"github.com/pion/mediadevices"
-	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
-	_ "github.com/pion/mediadevices/pkg/codec/vpx"      // This is required to register VP8/VP9 video encoder
+	_ "github.com/pion/mediadevices/pkg/codec/openh264"    // This is required to register h264 video encoder
+	_ "github.com/pion/mediadevices/pkg/codec/vpx"         // This is required to register VP8/VP9 video encoder
+	_ "github.com/pion/mediadevices/pkg/driver/camera"     // This is required to register camera adapter
+	_ "github.com/pion/mediadevices/pkg/driver/microphone" // This is required to register microphone adapter
```

Comment:
Audio is not used in this example. It can be omitted.

Reply:
my bad, copy-paste error lol